site/content/docs/5.2/forms/overview.md

```diff
@@ -77,6 +77,14 @@ Form text below inputs can be styled with `.form-text`. If a block-level element
 
 Inline text can use any typical inline HTML element (be it a `<span>`, `<small>`, or something else) with nothing more than the `.form-text` class.
 
+<!-- Boosted mod : design callout -->
+{{< ods-incompatibility-alert >}}
+This inline version of the form text input component can not be used because it does not respect the Orange Design System specifications. The form text input label should always be placed above the input field.
```

Comment:
OK

Reply:
New message:
This form variant, with label that is not above the input field, should not be used because it does not respect the Orange Design System specifications.

For the correct design, please refer to [Forms](https://system.design.orange.com/0c1af118d/p/88ab5b-forms/b/599459) and [Pages](https://system.design.orange.com/0c1af118d/p/20500e-form/b/16bb53) on the Orange Design System website.

TODO:
- add an exemple in DSM